### DISCUSSION TITLE
text_to_sql_pipeline: reference multiple tables?

### DISCUSSION THREAD

**Discussion by antonio-lamberti** [open-webui/pipelines#308] [General]
Hi!

I have import the text to sql pipeline, but there is no way to give the model access to more tables (or ideally, all tables!).
Is this a bug?

https://github.com/open-webui/pipelines/blob/main/examples/pipelines/rag/text_to_sql_pipeline.py

![image](https://github.com/user-attachments/assets/2f620381-de78-4d68-b0fb-4155f3a02023)

**Comment by Urvesh71** [comment]
I am really curious to know how to integrate this particular pipeline in Open web UI and enable it? I successfully verified the Pipeline connection but in 'Pipeline' section, when I upload this text_to_sql_pipeline, it shows me No Valves. Could you please explain me how you enabled it in your Open Web UI?

**Reply by antonio-lamberti** [reply]
Something is wrong if you don't even see the imported pipeline in the list. Look at the log from pipeline and owbui instances in docker for a clue

**Reply by Urvesh71** [reply]
I would like to ask you if you have configured the Postgresql database connection or not? If yes, Could you please write me steps for the guideline?

I could see Other Pipelines like Wikipedia so Pipeline could be detected. But Specificaly, I could not enable the text_to_sql_pipeline
